web/scss/partials/_header.scss

```diff
@@ -0,0 +1,136 @@
+.header {
+  top: 0; left: 0;
+
+  background-color: $lbry-white;
+  border-bottom: 1px solid $lbry-gray-1;
+  font-size: 1rem;
+  padding-top: 1rem;
+  position: fixed;
+  width: 100%;
+  z-index: 2;
+
+  > .inner-wrap {
+    display: flex;
+  }
+}
+
+.header--dark {
+}
+
+.header--light {
+}
+
+
+
+.header__download {
+  height: 100%;
+  line-height: 3;
+}
+
+.header__logo {
+  @include hide-text;
+  width: 8rem; height: 3rem;
+
+  background-image: url("../img/text-logo.svg");
+  background-position: 3rem center;
+  background-repeat: no-repeat;
+  background-size: 5rem;
+  display: block;
+  margin-right: 2.5rem;
+  position: relative;
+
+  &::before {
+    width: 3rem; height: 3rem;
+
+    background-image: url("../img/logo.svg");
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: 2.5rem;
+    content: "";
+    position: absolute;
+  }
+}
+
+
+
+.header__navigation {
+  display: inline-flex;
+  flex: 1;
+}
+
+.header__navigation-parent {
+  > li {
+    cursor: default;
+    height: 100%;
+    line-height: 3;
+    padding-bottom: 1rem;
+    position: relative;
+
+    &::after {
+      width: 100%; height: 1px;
+      bottom: -1px; left: 0;
+
+      content: "";
+      position: absolute;
+      z-index: 1;
+    }
+  }
+
+  &:not(:last-of-type) {
+    padding-right: 1rem;
+  }
+
+  &:not(:hover) {
+    > li::after {
+      background-color: transparent;
+    }
+
+    > .header__navigation-children-wrap {
+      display: none;
+    }
+  }
+
+  &:hover {
+    > li::after {
+      background-color: $lbry-teal-3;
+    }
+  }
+}
+
+.header__navigation-children-wrap {
+  top: 5rem; left: 0;
+
+  background-color: $lbry-white;
+  border-top: 1px solid $lbry-gray-1;
+  padding-top: 2rem;
+  padding-bottom: 2rem;
+  position: absolute;
+  width: 100%;
+
+  &::after {
+    width: 100vw; height: calc(100vh - 5rem);
+    top: 5rem; left: 0;
+
+    background-color: rgba($lbry-black, 0.3);
+    content: "";
+    pointer-events: none;
+    position: absolute;
+    z-index: -1;
+  }
+}
+
+.header__navigation-children {
+  display: flex;
+  flex-wrap: wrap;
+  position: relative;
+
+  li {
+    &:not(:last-of-type) {
```

Comment:
Not necessary to fix and possibly worth a discussion in chat with app team generally on pros/cons, but I've found `li + li { margin-top: 1rem; }` to be a mildly preferable approach.

Reply:
`margin-top` is not preferable to me because the flow of content is top-down, not down-up. In certain circumstances I'll use it but it doesn't need to be here.